src/Util/Exporter.php

```diff
@@ -0,0 +1,49 @@
+<?php declare(strict_types=1);
+/*
+ * This file is part of PHPUnit.
+ *
+ * (c) Sebastian Bergmann <sebastian@phpunit.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace PHPUnit\Util;
+
+use PHPUnit\TextUI\Configuration\Registry;
+use SebastianBergmann\Exporter\Exporter as OriginalExporter;
+
+/**
+ * @no-named-arguments Parameter names are not covered by the backward compatibility promise for PHPUnit
+ */
+final class Exporter
+{
+    private static ?OriginalExporter $exporter = null;
+
+    public static function export(mixed $value): string
+    {
+        return self::exporter()->export($value);
+    }
+
+    public static function shortenedRecursiveExport(array &$data): string
```

Comment:
```suggestion
    public static function shortenedRecursiveExport(array $data): string
```

Reply:
I would rather not have a different signature in PHPUnit's wrapper for the method from `Exporter`.

Reply:
AFAIK in exporter the `&` is needed to prevent recursion, here it is not needed, thus IMO, `&` is useless and only confusing here.